src/StaticAnalysis/ExecutableLinesFindingVisitor.php

```diff
@@ -165,6 +165,14 @@ private function getLines(NodeAbstract $node, bool $fromReturns): array
         }
 
         if ($node instanceof BinaryOp) {
+            if ($node instanceof BinaryOp\Concat &&
+                (
+                    $node->left instanceof Node\Expr\Variable ||
+                $node->right instanceof Node\Expr\Variable
+                )) {
+                return [$node->right->getStartLine()];
```

Comment:
```suggestion
                return [$this->getNodeStartLine($node->right)];
```

will fix start line for everything, string, nowdoc, ... never use fixed start/end line on a generic node

Reply:
Done :+1: